docs/moderators.md

```diff
@@ -94,3 +94,24 @@ Adding protected rooms on the fly is as easy as `!mjolnir rooms add <room alias>
 which are protected with `!mjolnir rooms`, and remove a room with `!mjolnir rooms remove <room alias>`. Note
 that rooms which are listed in the config may be protected again when the bot restarts - to remove these rooms
 permanently from protection, remove them from the config.
+
+## Trusted Reporters
+
+Mjolnir has an (optional) system in which it will poll Synapse for new reports, and when it sees sufficient
+(configurable limit) amounts of reports from trusted (configurable list) users on an given report, it will
+take a configurable action, such as redacting the message.
```

Comment:
The parenthesis read a bit choppy to my non-native eyes. Would the following work?

```suggestion
Mjolnir has an (optional) system in which it will poll Synapse for new reports, and when it sees sufficient amounts of reports from trusted users on an given report, it will take a configurable action, such as redacting the message.

Both the users to trust and the threshold to reach before taking action are configurable.
```